http.go

```diff
@@ -308,6 +349,12 @@ func (rnr *httpRunner) Run(ctx context.Context, r *httpRequest) error {
 			return err
 		}
 		r.setContentTypeHeader(req)
+
+		// Override useCookie
+		if r.useCookie == nil && rnr.useCookie != nil && *rnr.useCookie {
+			r.useCookie = rnr.useCookie
+		}
```

Comment:
httpRunner setting is enabled only if the useCookie option is unset in the req section